### ISSUE TITLE
Upgrading to 1.1.1 iOS app crash

### ISSUE THREAD

**Issue by adeem** [exponea/exponea-react-native-sdk#70]
I am not able to run the app because of upgrading to 1.1.1

```
error: Error: While trying to resolve module `react-native-exponea-sdk` from file `/src/core/app/hooks/useExponea/index.ts`, the package `/node_modules/react-native-exponea-sdk/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/node_modules/react-native-exponea-sdk/lib/index.js`. Indeed, none of these files exist:

  * /node_modules/react-native-exponea-sdk/lib/index.js(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.svg|.native.svg|.svg)
  * /node_modules/react-native-exponea-sdk/lib/index.js/index(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.svg|.native.svg|.svg)
    at DependencyGraph.resolveDependency (/node_modules/metro/src/node-haste/DependencyGraph.js:311:17)
    at Object.resolve (/node_modules/metro/src/lib/transformHelpers.js:129:24)
    at resolve (/node_modules/metro/src/DeltaBundler/traverseDependencies.js:396:33)
    at /node_modules/metro/src/DeltaBundler/traverseDependencies.js:412:26
    at Array.reduce (<anonymous>)
    at resolveDependencies (/node_modules/metro/src/DeltaBundler/traverseDependencies.js:411:33)
    at processModule (/node_modules/metro/src/DeltaBundler/traverseDependencies.js:140:31)
    at async addDependency (/node_modules/metro/src/DeltaBundler/traverseDependencies.js:230:18)
    at async Promise.all (index 3)
    at async processModule (/node_modules/metro/src/DeltaBundler/traverseDependencies.js:198:5)
```

can you help me out?

**Comment by adam1929** [comment]
Hi @adeem this problem has been fixed in 1.1.1-1 version. SDK has been released but build was invalid. Patched version has correct setup

**Comment by adeem** [comment]
thanks and confirm that its working now with 1.1.1-1 👍
